shaders/SelectDistinctFromBackgroundColor.shader

```diff
@@ -0,0 +1,17 @@
+shader_type canvas_item;
+
+uniform vec4 bg_colour : hint_color;
+
+uniform vec4 main_colour : hint_color;
+uniform vec4 sec_colour : hint_color;
```

Comment:
```suggestion
uniform vec4 backgroundColour : hint_color;

uniform vec4 mainColour : hint_color;
uniform vec4 secondaryColour : hint_color;
```
for consistent naming with other shaders

Reply:
You didn't commit a fix for this?

Reply:
Whops, didn't wait for the console output of git warning me about unpulled changes :D